examples/landmark_detection/example_scan.py

```diff
@@ -0,0 +1,218 @@
+# %%
+
+import numpy as np
+import pandas as pd
+
+
+from pathlib import Path
+
+from giskard_vision.scanner.scanner_vision import Scanner
+from giskard_vision.landmark_detection.models.wrappers import OpenCVWrapper
+from giskard_vision.landmark_detection.dataloaders.loaders import DataLoader300W
+
+from face_alignment import FaceAlignment, LandmarksType
+
+from giskard_vision.landmark_detection.dataloaders.wrappers import (
+    CroppedDataLoader,
+    ResizedDataLoader,
+    ColoredDataLoader,
+    BlurredDataLoader,
+    FilteredDataLoader,
+    HeadPoseDataLoader,
+    EthnicityDataLoader,
+    CachedDataLoader,
+)
+from giskard_vision.landmark_detection.tests.performance import NMEMean
+from giskard_vision.landmark_detection.tests.base import Test, TestDiff
+from giskard_vision.landmark_detection.marks.facial_parts import FacialParts
+from giskard_vision.scanner.scanner_vision import Scanner
+
+from giskard.scanner.issues import Issue, IssueLevel, IssueGroup
+
+# %%
+
+
+class FacialPartsDetector:
```

Comment:
should inherit from: https://github.com/Giskard-AI/giskard/blob/1882fb6fa617216266d9a8b1fd7fd6307f43a365/giskard/scanner/registry.py#L11-L14